sys/include/suit/v3/handlers.h

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (C) 2019 Koen Zandberg
+ *               2019 Kaspar Schleiser <kaspar@schleiser.de>
+ *               2019 Inria
+ *               2019 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+/**
+ * @ingroup     sys_suit_v3
+ * @brief       SUIT v3 manifest handlers
```

Comment:
Here too, how about being precise in the comments/doc and specify draft-ietf-suit-manifest-03 instead of just "v3" ?

Reply:
Fixed